src/app/styles/index.scss

```diff
@@ -12,4 +12,83 @@
 @font-face {
     font-family: Rubik;
     src: url(fonts/Rubik-Regular.ttf);
-}
\ No newline at end of file
+}
+
+@layer components {
+   .h1 {
+      @apply font-rubik font-normal text-8xl
+   }
+
+   .h2 {
+      @apply font-rubik font-semibold text-[50px]
+   }
+
+   .h3 {
+      @apply font-rubik font-medium text-5xl
+   }
+
+   .h4 {
+      @apply font-rubik font-normal text-[46px]
+   }
+
+   .h5 {
+      @apply font-rubik font-normal text-4xl
+   }
+
+   .h6 {
+      @apply font-rubik font-light text-2xl
+   }
+
+   .primary {
+      @apply font-rubik font-normal text-xl
+   }
+
+   .primary-bold {
+      @apply font-rubik font-semibold text-[22px]
+   }
+
+   .secondary {
+      @apply font-rubik font-light text-lg
+   }
+
+   .secondary-bold {
+      @apply font-rubik font-medium text-lg
+   }
+
+   .additional {
+      @apply font-rubik font-normal text-base
+   }
+}
+
+
+.catalog-icon:hover circle {
+    fill: #00BCD4;
+ }
+
+ .present-icon:hover path:nth-child(1){
+    fill: #4495ff;
+ }
+
+ .present-icon:hover path:nth-child(2), .present-icon:hover path:nth-child(5){
```

Comment:
Please create nested structure of classes.
.present-icon {
    &:hover {
        ...
    }
}